src/modules/new-request-form/fields/cc-field/useTagsInputContainer.ts

```diff
@@ -90,12 +90,14 @@ export function useTagsInputContainer({
     const target = e.target as HTMLInputElement;
     const tag = target.value;
 
+    console.log(e.key);
```

Comment:
Nit: debug leftover?

Reply:
was still in draft! I made the keys consistent now